packages/@ember/routing/route.ts

```diff
@@ -1256,11 +1256,11 @@ class Route<Model = unknown> extends EmberObject.extend(ActionHandler, Evented)
     if (ENV._NO_IMPLICIT_ROUTE_MODEL) {
       return;
     }
-    deprecateUntil(
-      `The implicit model loading behavior for routes is deprecated. ` +
-        `Please define an explicit model hook for ${this.fullRouteName}.`,
-      DEPRECATIONS.DEPRECATE_IMPLICIT_ROUTE_MODEL
-    );
+    // deprecateUntil(
```

Comment:
this is a good deprecation -- why comment it out? 😅

Reply:
restored